Gems/AtomLyIntegration/EditorModeFeedback/Assets/Shaders/EditorModeBufferCopy.azsl

```diff
@@ -13,7 +13,7 @@ PSOutput MainPS(VSOutput IN)
     PSOutput OUT;
 
     // Copy the input buffer over to the output buffer
-    OUT.m_color.rgb = PassSrg::m_framebuffer.Sample(PassSrg::LinearSampler, IN.m_texCoord);
+    OUT.m_color = PassSrg::m_framebuffer.Sample(PassSrg::LinearSampler, IN.m_texCoord);
```

Comment:
While this fixes a warning, it's not doing the same, the same behavior would be:

```suggestion
    OUT.m_color.rgb = PassSrg::m_framebuffer.Sample(PassSrg::LinearSampler, IN.m_texCoord).rgb;
```

Reply:
Alright, if `a` is not written anywhere else, you should drop my suggestion and keep your change.